src/main/java/com/terraformersmc/terrestria/init/TerrestriaBlocks.java

```diff
@@ -198,8 +202,13 @@ public static void init() {
 		BASALT_GRASS_BLOCK = TerrestriaRegistry.register("basalt_grass_block", new BasaltGrassBlock(BASALT_DIRT, () -> BASALT_GRASS_PATH, FabricBlockSettings.copy(Blocks.GRASS_BLOCK).breakByTool(FabricToolTags.SHOVELS, 0).build()));
 		BASALT_GRASS_PATH = TerrestriaRegistry.register("basalt_grass_path", new TerraformGrassPathBlock(BASALT_DIRT, FabricBlockSettings.copy(Blocks.GRASS_PATH).breakByTool(FabricToolTags.SHOVELS, 0).build()));
 		BASALT_PODZOL = TerrestriaRegistry.register("basalt_podzol", new Block(FabricBlockSettings.copy(Blocks.PODZOL).breakByTool(FabricToolTags.SHOVELS, 0).build()));
+		BASALT_FARMLAND = TerrestriaRegistry.register("basalt_farmland", new TerraformFarmlandBlock(FabricBlockSettings.copy(Blocks.FARMLAND).materialColor(MaterialColor.BLACK).breakByTool(FabricToolTags.SHOVELS, 0).build(), BASALT_DIRT));
 		BASALT = StoneBlocks.register("basalt", MaterialColor.BLACK);
 
+		HoeUtil.putTillable(BASALT_DIRT, BASALT_FARMLAND.getDefaultState());
```

Comment:
Maybe in the style of fabric API this could be called TillableBlockRegistry or something like that.